lib/DelUserCmd.pm

```diff
@@ -0,0 +1,25 @@
+package DelUserCmd;
+use strict;
+use warnings FATAL => 'all';
+use v5.18;
+use parent qw(Command);
+
+use UserService;
+
+sub execute {
+    my $self = shift;
+    my @username = $self->parameters();
+    my $user_service = UserService->new();
```

Comment:
Try to inject UserService as a param of the method, with this approach you could mock the method delete_user and isolate the code that you must test.